NOTES.UNIX

```diff
@@ -89,3 +90,22 @@
 
     $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
         '-Wl,--enable-new-dtags,-rpath,$(LIBRPATH)'
+
+ It might be worth noting that some/most ELF systems implement support
+ for relative runtime search path, relative to directory containing
```

Comment:
Transported from #6560: s/for relative runtime search path, relative to directory containing/for runtime search path relative to directory containing/